src/assets/styles/modules/responsive-images.scss

```diff
@@ -0,0 +1,31 @@
+/* ================ Responsive images ================ */
+.responsive-image__wrapper {
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  margin: 0 auto;
+
+  .no-js & {
+    display: none;
+  }
+
+  &:before {
+    content: '';
+    position: relative;
+    display: block;
+  }
+
+  img {
+    display: block;
+    width: 100%;
+  }
+}
```

Comment:
Perhaps we should also include the following for scenarios where we might want to use `object-fit` for our images:

```css
.responsive-image__wrapper--full-height {
  height: 100%;
}
```

Which we can then pass to our snippet using the `wrapper_class` variable.